achievements.lua

```diff
@@ -168,41 +321,202 @@ function achievements.initialize(gamedata, prevent_debug)
 		ach.granted_at = achievements.granted[ach.id] or false
 	end
 
+	load_images()
 	playdate.file.mkdir(get_achievement_folder_root_path(gamedata.gameID))
 	export_data()
-	copy_images_to_shared()
+	copy_images_to_shared(gamedata.gameID, (tonumber(gamedata.buildNumber) or 0))
 
 	print("files exported to /Shared")
 	print("Achievements have been initialized!")
 	print("------")
 end
 
+--[[ Achievement Drawing & Animation ]]--
```

Comment:
This section seems broadly solid from a cursory inspection, though I'll note I haven't taken the time to test it firsthand.